### PR TITLE
Extract media (List/Player) modules

### PR DESCRIPTION
**Pull request by mhdabbaghy**
#86

### CODE REVIEW

File: app/src/main/java/ir/mehdiyari/krypt/ui/KryptApp.kt [hunk 1]
```diff
@@ -48,7 +48,7 @@ fun KryptApp(
                 when (it) {
                     R.string.add_media -> {
                         kryptAppState.navController.navigateToMedia(
-                            MediaViewAction.PICK_MEDIA
+                            ir.mehdiyari.krypt.mediaList.MediaViewAction.PICK_MEDIA
```
Comment (mehdiyari) [inline app/src/main/java/ir/mehdiyari/krypt/ui/KryptApp.kt:51]: Please import MediaViewAction, here and in the rest of the changes

**Comment by mehdiyari** [comment]
#49

**Comment by mehdiyari** [comment]
Please resolve the conflicts and comments @mhdabbaghy. 
Also, did you fix the back stack bug in the media screen?

**Comment by mhdabbaghy** [comment]
@mehdiyari Conflicts are resolved and imports are optimized now
TBH despite a tested that and reproducing that in the first place but I forgot that to fix it, I'll fix it today

**Comment by mehdiyari** [comment]
> 

Thanks, MHD, Please create another branch for that fix, I will merge this PR today.

**Pull request merged**